src/data/arena-tag.ts

```diff
@@ -743,8 +750,10 @@ export abstract class EntryHazardTag extends SerializableArenaTag {
   /**
    * The current number of layers this tag has.
    * Starts at 1 and increases each time the trap is laid.
+   * Should not be accessed by anything other than this class (with exceptions for arena flyout-related code).
+   * @protected
```

Comment:
`@protected` and `@private` are not valid TSDocs (but `@public` is for some reason...).
```suggestion
   * The current number of layers this tag has. \
   * Starts at `1` and increases each time the trap is laid.
   * @privateRemarks
   * Should not be modified by anything other than this class or its subclasses. \
   * It cannot be marked as `protected` for serialization reasons.
```